data/ignore_ids_safety_scan.json

```diff
@@ -412,6 +412,7 @@
             "py2": {
             },
             "py3": {
+                "51159":"cryptography>38.0.1 does not exist yet"
```

Comment:
Since this is the only vulnerability present in `inference` for `Pytorch` lets keep it as it is. In the future, we can make a key for `inference_graviton`

Reply:
/agreed